spec/features/merchants/destroy_spec.rb

```diff
@@ -1,11 +1,17 @@
 require 'rails_helper'
 
-RSpec.describe "As a visitor" do
-  describe "When I visit a merchant show page" do
+RSpec.describe "As a merchant" do
```

Comment:
Updated this to reflect that no anyone can destroy a merchant, only that merchant